taker-frontend/src/components/LiquidationPrice.ts

```diff
@@ -0,0 +1,16 @@
+function quantoLiquidationPrice(leverage: number, quantity: number, price: number) {
+    return (price * (1 / leverage));
```

Comment:
📣 Wait a minute, this is a different formula! And it's correct. In the `daemon` I had:

```
bankruptcy_price_shift = initial_margin / (multiplier * n_contracts)
```

but this is equal to

```
bankruptcy_price_shift = ((n_contracts * initial_price * multiplier) / leverage) / (multiplier * n_contracts)
                       = initial_price / leverage
```

which means we don't need the number of contracts to calculate the quanto liquidation price!!! So we can calculate it using the same components we use for the inverse one. Meaning we can compute all this in the `daemon`.

Sorry for the extra work. I had a suspicion that my equation wasn't "right", but we didn't see it at the time.

Reply:
haha... I could have noticed that 😅 
How boring is that if we can compute everything in the daemon...

Reply:
actually, we still need the leverage from the ui for that, which mean we would need to compute a _unit_ value and finish it in the frontend.

Reply:
> actually, we still need the leverage from the ui for that, which mean we would need to compute a _unit_ value and finish it in the frontend.

We already did that for inverse CFDs, right? We could also pre-compute all possible values in the daemon based on the valid leverages 😛

Reply:
Oh... this was never working correctly ... we always computed it for leverage 2 but did not change it for leverage 1 or 3 😅

Reply:
I'm obviously biased here, but given we have to do some computations in the frontend anyways, I would be in favor of going the approach of this PR, i.e. compute the liquidation price fully in the frontend.

Reply:
I'm not a huge fan if it can be avoided, because it's duplicated logic and it's written in TS haha. I won't block the PR nevertheless.

Reply:
Our conclusion in the past was: Don't do complex calculations in the UI, especially avoid divisions (because that might lead to loss in JS). We usually went for using the `lot` and `leverage` as multiplier, but we avoided divisions. 
If we can apply this rule here as well I think it will be better than moving more complex calculations into the UI now. 
Having duplicated calculations in rust and TS is really not encouraged.

Reply:
I don't think this is a critical part, if we have loss with calculating the liquidation price there should be not any disagreement between the components and for the user I don't think it will matter much. But nonetheless I don't think it's a good idea to keep changing the paradigms we introduce hastily because it makes the code harder to reason about. If we decide to merge this please at least add a ticket so we can clean this up afterwards, i.e. move it into the daemon, because I think it should be there unless we record a dedicated decision to do this different.